antarest/launcher/service.py

```diff
@@ -252,7 +252,7 @@ def run_study(
         self.launchers[launcher].run_study(
             study_uuid,
             job_uuid,
-            str(solver_version),
+            solver_version,
```

Comment:
I think here we should change the signature to accept a `SolverVersion` object, to clarify what kind of version it is

Reply:
I used `SolverVersion` and ultimately made a if/else to parse it as a string inside the `study.antares` file